src/Request.php

```diff
@@ -164,4 +136,77 @@ protected function throwServerError($body, $httpStatusCode)
             ErrorCode::SERVER_ERROR,
             $httpStatusCode);
     }
+
+    protected function getRequestHeaders()
+    {
+        $uaHeader = array(
+            'User-Agent' => $this->constructUa()
+        );
+
+        $headers = array_merge(self::$headers, $uaHeader);
+
+        return $headers;
+    }
+
+    protected function constructUa()
+    {
+        $ua = 'Razorpay/v1 PhpSdk/' . Api::VERSION;
```

Comment:
@shashankkumar I've been thinking of including PHP Version here as well, to help us gauge usage. Thoughts?